packages/nuxt/src/app/composables/ssr.ts

```diff
@@ -56,3 +57,10 @@ export function prerenderRoutes (path: string | string[]) {
   const paths = toArray(path)
   appendHeader(useRequestEvent(), 'x-nitro-prerender', paths.map(p => encodeURIComponent(p)).join(', '))
 }
+
+export const setInterval = import.meta.client ? window.setInterval : () => {
+  throw createError({
+    statusCode: 500,
+    message: 'setInterval should not be used on server side, please wrap it with `onBeforeMount` or `onMounted` Vue Lifecycle hooks.'
```

Comment:
Happy to update it if you know how to do it 😄 

So far it looks like this:
![CleanShot 2024-01-17 at 14 59 42@2x](https://github.com/nuxt/nuxt/assets/904724/95b6fbc3-34f1-491a-a999-89d27013a09e)

Would be nice also to get the source map to link back here to `app.vue` instead of `app.js`, but same, not sure how to do this.

Can be handled in another PR to move one faster though

Reply:
Yeah, let's go with the initial version and possibly raise an issue for improvements 👍

Reply:
You might try `Error.captureStackTrace` if it can help (https://v8.dev/docs/stack-trace-api)

Reply:
Hum does not help:

![CleanShot 2024-01-17 at 15 27 59@2x](https://github.com/nuxt/nuxt/assets/904724/8959f99a-e089-4ed9-9811-0f1cbd9d70c2)

I think it's fine in the current step since the message is quite explicit.